src/menu/chapter-thumbnail-menu-button.js

```diff
@@ -201,6 +188,8 @@ class ChapterThumbnailMenuButton extends VjsMenuButton {
 
 ChapterThumbnailMenuButton.prototype.controlText_ = 'Chapters';
 
+videojs.registerComponent('ChapterThumbnailMenuButton', ChapterThumbnailMenuButton);
```

Comment:
Publicly registering components so that people can override their implementation.

Also, update the generation of components to pull from the previously registered components (This is a weird pattern, but I am not sure how else the framework will allow this feature).